internal/tests/api/with-ns-registered/minipool_count_test.go

```diff
@@ -0,0 +1,39 @@
+package with_ns_registered
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	hdtesting "github.com/nodeset-org/hyperdrive-daemon/testing"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	nodeWalletAddressString string = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
```

Comment:
This is already loaded in `main_test`, don't need this here.

Reply:
updated